src/Microsoft.AspNetCore.SignalR.Core/Hub.cs

```diff
@@ -55,20 +67,34 @@ public IGroupManager Groups
             }
         }
 
+        /// <summary>
+        /// Called when the hub is connected to.
```

Comment:
Called when **a new connection is established with** the hub ~~is connected to~~.